src/test/java/ibm/jceplus/junit/openjceplus/TestAESCopySafe.java

```diff
@@ -38,7 +38,7 @@ public static void main(String[] args) throws Exception {
     //
     //
     public static Test suite() {
-        TestSuite suite = new TestSuite(TestAESGCMCopySafe.class);
+        TestSuite suite = new TestSuite(TestAESCopySafe.class);
```

Comment:
Should we remove this method and `main` as well?

Reply:
As part of the work done in https://github.com/IBM/OpenJCEPlus/pull/252 I did remove all `main` methods and migrated each test to Junit 5. Once this PR has been approved and merged is it OK that i do that work in that PR? I will need to resolve differences anyway in order to update the Junit 5 PR for the test name change being made here also.